public/app/core/constants.ts

```diff
@@ -12,3 +15,18 @@ export const LS_PANEL_COPY_KEY = 'panel-copy';
 export const PANEL_BORDER = 2;
 
 export const EDIT_PANEL_ID = 23763571993;
+
+export const DEPRECATED_PANELS: Record<string, (panel: PanelModel) => string> = {
+  singlestat: (panel: PanelModel) => {
+    // If 'grafana-singlestat-panel' exists, move to that
```

Comment:
See: https://github.com/grafana/singlestat-panel

For some reason, this does not seem to keep all the settings, but it does migrate :man_shrugging:

Reply:
and now you can have:  
![image](https://user-images.githubusercontent.com/705951/112259703-f78fb100-8c25-11eb-88e0-3ad382ff56d0.png)